### PR TITLE
Todo project by Vladimir Bykov

### PR DESCRIPTION
**Pull request by vladjnbykov**
Link to my deployed project:
https://youthful-gates-351c5c.netlify.app/

My reflections on how this week's project turned out:
It took a while to get some grasp on the concept. Ultimately it went not bad, the application works as intended.

Things I'd like to have clarified or explained in more detail:
Still problems with understanding when I need to use components, reducers and selectors

### CODE REVIEW

File: code/src/components/todoList.css [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+    transform: perspective(1000px) rotateX(-77deg)
+  }
+  
+  .all-done:hover {
+    content: 'sure?';
+    color: #020355;
+    font-weight:900;
+    background-color: rgba(250, 4, 4, 0.137);
+    cursor: pointer;
+  }
```
Comment (Bellsibub) [inline code/src/components/todoList.css:129]: This was sooo cool! 🚀 🤯

File: code/src/reducers/todos.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+    },
+
+    allDone: (store) => {
+      const Done = store.items.map((todo) => {
+        return {
+          ...todo,
+          isComplete: true
+        }
+      })
+      store.items = Done
```
Comment (Bellsibub) [inline code/src/reducers/todos.js]: Just curious, why did you choose to make this `const` with a capital letter? 🤔

**Review by Bellsibub — APPROVED** [review]
Veeery nice project! 👍  Loved the epic transitions on the hover for the All Done button! 🎉 

Couldn't find any issues that I omit to a change request. So I approved it 😉 
However I did leave some comments and questions on specific parts down below ⏬ 

No matter, I consider this project done! 💯

**Comment by vladjnbykov** [comment]
Hi Bella,

Thank you very much for nice comments!
I have now fixed the name of the variable. The reason for capital letter 
could be that I was too happy that the project was finally done, so, the 
name :).

Happy Valborg!
Vlad


------ Originalmeddelande ------
Från: "Bellsibub" ***@***.***>
Till: "Technigo/project-todos" ***@***.***>
Kopia: "Vladimir JN Bykov" ***@***.***>; "Author" 
***@***.***>
Skickat: 2021-04-28 17:34:27
Ämne: Re: [Technigo/project-todos] Todo project by Vladimir Bykov (#159)

File: code/src/components/TodoList.js [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+  return (
+    <div>
+      {items.map((todo) => (
+        <div key={todo.id} className="todo-items">
+          <div className="todo-item">
+            <input
+              className="checkbox"
+              type="checkbox"
+              checked={todo.isComplete}
+
```
Comment (KarolCislowski) [inline code/src/components/TodoList.js:28]: FIGHT WITH LAZYNESS!! 🔫 add REQUIRED to make your to-do free from empty tasks!! 🎖️ 😆

File: code/src/components/TodoList.js [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+            <div className="counter">
+              {todo.isComplete ? count : count += 1}
+            </div>
+
+          </div>
+
+          <div className="time">
+                        task created: {todo.timeCreated}
+          </div>
+
```
Comment (KarolCislowski) [inline code/src/components/TodoList.js:45]: It is no need to use so many empty lines. Put them when you need to separate different logic blocks.

**Review by KarolCislowski — APPROVED** [review]
Nice app 😸 

You asked about components etc. It is a complex subject so if you need to have it clarified you should book a 1:1 session with one of the coaches 😸

**Pull request closed without merging**